ironfish/src/mining/director.ts

```diff
@@ -363,12 +363,18 @@ export class MiningDirector {
       totalTransactionFees += transactionFee
     }
 
+    const newSequence = blockHeader.sequence + 1
+
     const minersFee = await this.strategy.createMinersFee(
       totalTransactionFees,
-      blockHeader.sequence + 1,
+      newSequence,
       this._minerAccount.spendingKey,
     )
 
+    this.logger.info(
```

Comment:
Opinions on whether this log is too aggressive/frequent at `info` level? I think it'd be useful to see if there are any issues with generating a transaction for the wrong account right now, but we could bump it to `debug` later.

Reply:
I would advocate for changing this to debug sine the info is for debugging miner problems.

Right now we don't have any constant logs, but this will fill the terminal and make it hard to see what's happening in the node.

Reply:
Changed this -- I'm interested in brainstorming a way we can still save these logs without dumping them to the terminal. It's one of a category of logs where 99% of the time it's not useful, sometimes you retroactively would have wanted to enable it. For example, if someone mined a block on the main chain and is wondering why it's not showing up in their account.